src/main/java/util/xcmailr/XcMailrApi.java

```diff
@@ -25,58 +30,111 @@
     public final static String TEMPORARY_CONFIG_FILE_PROPERTY_NAME = "xcmailr.temporaryConfigFile";
 
     /**
-     * Retrieves the instance of xcmailr configuration for the current thread.
+     * Retrieves the {@link XcMailrConfiguration} instance of the current thread.
      * 
-     * @return the configuration instance for the current thread
+     * @return the XcMailrConfiguration instance of the current thread
      */
     public static XcMailrConfiguration getConfiguration()
     {
         // the property needs to be a valid URI in order to satisfy the Owner framework
         if (null == ConfigFactory.getProperty(TEMPORARY_CONFIG_FILE_PROPERTY_NAME))
         {
-            ConfigFactory.setProperty(TEMPORARY_CONFIG_FILE_PROPERTY_NAME, "file:this/path/should/never/exist/noOneShouldCreateMe.properties");
+            ConfigFactory.setProperty(TEMPORARY_CONFIG_FILE_PROPERTY_NAME,
+                                      "file:this/path/should/never/exist/noOneShouldCreateMe.properties");
         }
         return CONFIGURATION.computeIfAbsent(Thread.currentThread(), key -> {
             return ConfigFactory.create(XcMailrConfiguration.class);
         });
     }
 
-    public static void createTemporaryEmail(String email)
+    /**
+     * Creates a temporary e-mail. <br>
+     * The validity period is determined from the configuration.
+     * 
+     * @param email
+     *            the e-mail address <br>
+     *            The domain of the e-mail must match the XcMailr's available domains.
+     */
+    public static String createTemporaryEmail(String email)
     {
-        String url = getConfiguration().url() + "/create/temporaryMail/" + getConfiguration().apiToken() + "/" + email + "/"
-                     + getConfiguration().temporaryMailValidMinutes();
-        HttpUrl.Builder builder = HttpUrl.parse(url).newBuilder();
-        Response response = callXcMailr(builder.build());
+        final String url = getConfiguration().url() + "/create/temporaryMail/" + getConfiguration().apiToken() + "/" + email + "/" +
+                           getConfiguration().temporaryMailValidMinutes();
+        final HttpUrl.Builder builder = HttpUrl.parse(url).newBuilder();
+        final Response response = callXcMailr(builder.build());
+        String responseBody = readResponse(response.body().byteStream()).toString();
 
         Assert.assertNotNull("XcMailr not reachable", response);
-        Assert.assertEquals("Temporary Email could not be created", 200, response.code());
-        LOGGER.debug("Email created: \"" + email + "\"");
+        Assert.assertEquals("Temporary Email could not be accessed", 200, response.code());
```

Comment:
We really create a temporary e-mail address here.
```suggestion
        Assert.assertEquals("Temporary Email could not be created", 200, response.code());
```